central/graphql/resolvers/compliance_test.go

```diff
@@ -62,8 +66,8 @@ func (s *ComplianceResolverTestSuite) TestDoesNotTruncateUnknownCollapseBy() {
 
 	truncatedResults, truncatedDomainMap, errorMessage := truncateResults(testResults, testDomainMap, testCollapseBy)
 	s.Empty(errorMessage)
-	s.Equal(testResults, truncatedResults)
-	s.Equal(testDomainMap, truncatedDomainMap)
+	protoassert.SlicesEqual(s.T(), testResults, truncatedResults)
+	compareResults(s.T(), testDomainMap, truncatedDomainMap)
```

Comment:
Is there a reason why we can not use `protoassert.MapEqual` here?

Reply:
good catch, I think I got compilation error on not comparable key but it's not the case now.